cupsfilters/fontembed/sfnt.c

```diff
@@ -517,7 +517,7 @@ _cfFontEmbedOTFGetTable(_cf_fontembed_otf_file_t *otf,
   const int idx = __cfFontEmbedOTFFindTable(otf, tag);
   if (idx == -1)
   {
-    *ret_len =- 1;
+    *ret_len = -1;
```

Comment:
The intention here is likely to make `ret_len` invalid rather than decrement it. There are other cases where `ret_len` is set to -1 or 0 when NULL is returned.